lib/metrics.js

```diff
@@ -100,7 +106,7 @@ function getMetrics() {
         .value();
 
     const totalTime = sum(_.values(summaries), 'total');
-    const lastCompleteCall = _.max(_.values(summaries), ['lastCall', 'endTime']).lastCall;
+    const lastCompleteCall = _.max(_.values(summaries), ['lastCall', 'startTime']).lastCall;
```

Comment:
I'm having trouble figuring out if this is changing and how... what's the point of this metric? Can we maybe add some code comments to help explain it. I had trouble with this last time and I'm even more confused now so it would be good to add some additional context.

Reply:
Can we remove the `lastCompleteCall` altogether? 
This metric would capture the last call made per each alias - get/set/merge etc... 
I don't find it very useful now that we see it on the timeline
The other place we might use it is in excel and it's pretty easy to derive from calls

Reply:
I am not opposed to removing it. But if it's helpful maybe just explain what it is. I think the part I'm struggling with is the change from `endTime` to `startTime`. Just sort of hard to tell what we are measuring or something from looking at the code so a comment would help.

Reply:
Updated to capture the endTime and added documentation